include/swift/AST/DiagnosticsFrontend.def

```diff
@@ -58,6 +58,14 @@ ERROR(error_unsupported_opt_for_target, none,
 ERROR(error_argument_not_allowed_with, none,
   "argument '%0' is not allowed with '%1'", (StringRef, StringRef))
 
+ERROR(error_option_requires_sanitizer, none,
+  "option '%0' requires a sanitizer to be enabled. Use -sanitize= to enable a"
+  " sanitzer", (StringRef))
+
+ERROR(error_option_missing_required_argument, none,
+  "option '%0' is missing a required argument. The required "
+  "argument %1", (StringRef, StringRef))
```

Comment:
:-/ I'm still unhappy with one of the diagnostic arguments being a whole phrase. How about `"option '%0' is missing a required argument (%1)"`? That way there's less of a grammatical restriction, and it's also more concise.